ci/vm.sh

```diff
@@ -59,12 +59,12 @@ make_disk() {
 }
 
 gen_metadata() {
-	local cprcmd hardware_id id
-	id=$(uuidgen)
+	local cprcmd hardware_id
 	hardware_id=$(uuidgen)
 	local class=$1
 	local slug=$2
 	local tag=$3
+	local id=$4
```

Comment:
This could use some disambiguation in naming: what kind of ID? is it a user id?

Alternatively, document the function inputs.

Reply:
fixed